src/network_models/power_flow_evaluation.jl

```diff
@@ -155,14 +235,28 @@ branch_aux_vars(::PFS.PSSEExporter) = DataType[]
 # Same for bus aux vars
 bus_aux_vars(data::PFS.ACPowerFlowData) =
     if data.calculate_loss_factors
-        [PowerFlowVoltageAngle, PowerFlowVoltageMagnitude, PowerFlowLossFactors]
+        [PowerFlowVoltageAngle, PowerFlowVoltageMagnitude,
+            PowerFlowBusActivePowerInjection, PowerFlowBusReactivePowerInjection,
```

Comment:
Could be useful to define
```julia
const InjectionWithdrawalAuxVars = [PowerFlowBusActivePowerInjection, PowerFlowBusReactivePowerInjection, PowerFlowBusActivePowerWithdrawals, PowerFlowBusReactivePowerWithdrawals]
```
so you can just do `InjectionWithdrawalAuxVars...` in cases like this